scripts/linters/general_purpose_linter.py

```diff
@@ -106,6 +97,18 @@
                    'in the format TODO(username): XXX. ',
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
```

Comment:
As per our discussion, change the BAD_PATTERN lint function to print the line number instead of moving it as a regex check.

Reply:
Done.